XenAdmin/Dialogs/HealthCheck/HealthCheckSettingsDialog.cs

```diff
@@ -316,7 +319,6 @@ private void credentials_TextChanged(object sender, EventArgs e)
 
         private void xsCredentials_TextChanged(object sender, EventArgs e)
         {
-            UpdateButtons();

```

Comment:
But would the user know that they need to press "Test credentials" in order for the OK button to become enabled?